### PR TITLE
OSOE-893: "Settings_Wrapper__General" should be changed to "Settings_Wrapper__Reload"

### PR DESCRIPTION
**Pull request by sarahelsaig**
Also Update OC preview, fix changes, and reorganize CreateModelMaybeAsync.

Fixes https://github.com/Lombiq/Helpful-Libraries/issues/279
Fixes https://github.com/Lombiq/Helpful-Libraries/issues/280

[OSOE-893](https://lombiq.atlassian.net/browse/OSOE-893)

[OSOE-893]: https://lombiq.atlassian.net/browse/OSOE-893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by MikeAlhayek** [comment]
https://github.com/OrchardCMS/OrchardCore/pull/16532

File: NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj [hunk 1]
```diff
@@ -19,49 +19,49 @@
...
-    <PackageReference Include="Lombiq.Privacy" Version="8.0.2-alpha.0.occ-245" />
-    <PackageReference Include="Lombiq.Privacy.Samples" Version="8.0.2-alpha.0.occ-245" />
-    <PackageReference Include="Lombiq.SetupExtensions" Version="6.0.2-alpha.0.occ-245" />
-    <PackageReference Include="Lombiq.Tests.UI.AppExtensions" Version="10.0.2-alpha.1.occ-245" />
-    <PackageReference Include="Lombiq.Tests.UI.Shortcuts" Version="10.0.2-alpha.1.occ-245" />
-    <PackageReference Include="Lombiq.UIKit" Version="7.0.2-alpha.0.occ-245" />
-    <PackageReference Include="Lombiq.VueJs" Version="6.0.1-alpha.3.occ-245" />
-    <PackageReference Include="Lombiq.VueJs.Samples" Version="6.0.1-alpha.3.occ-245" />
-    <PackageReference Include="Lombiq.Walkthroughs" Version="1.0.2-alpha.0.occ-245" />
+    <PackageReference Include="Lombiq.AuditTrailExtensions" Version="6.0.2-alpha.1.osoe-893" />
```
Comment (Piedone) [inline NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj:22]: 18300 has another driver change: https://github.com/OrchardCMS/OrchardCore/pull/16530

File: src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json [hunk 2]
```diff
@@ -91,13 +91,16 @@
       "name": "settings",
       // CDN should be turned off for tests, just as no other external resource should be used.
       "UseCdn": false,
-      // A breaking change as of OC 1.9, the SMTP e-mail provider is not turned on by default and has to be enabled.
+      // A breaking change as of OC 2.0, the SMTP e-mail provider is not turned on by default and has to be enabled.
       // Also host is null by default, so has to be explicitly specified.
       "SmtpSettings": {
         "IsEnabled": true,
         "DefaultSender": "sender@example.com",
         "Host": "localhost"
```
Comment (Piedone) [inline src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json]: These are now provided by `ConfigureHostingDefaults`.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json (hunk 2 above)
Comment (sarahelsaig) [reply src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json]: Are you sure this works? When I remove these lines I get this error:

Comment (sarahelsaig) [reply src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json]: And without `DefaultSender` it just says "connection refused".
image

Comment (Piedone) [reply src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json]: The config was incomplete, needed `IsEnabled` too.

**Review by Piedone — APPROVED** [review]

**Pull request merged**